### PR TITLE
CI: add workflow to purge jsdelivr cache

### PR DESCRIPTION
**Pull request by FranceBe**
This PR adds a workflow to manually purge JSDelivr cache when needed

### CODE REVIEW

**Comment by noe-charmet** [comment]
You can change the trigger mode to try running it from this PR

**Review by Francois-Gomis — APPROVED** [review]

File: .github/workflows/purge-js-delivr-cache.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Temporary trigger workflow on PR to test the job
+#on:
+#  workflow_dispatch:
+
+on:
+  pull_request:
+    branches: [master]
+
+jobs:
+  test-curl-action:
```
Comment (gautierlabarre) [inline .github/workflows/purge-js-delivr-cache.yml]: The action name could be : 'purge-jsdelivr-cache' or something like that

File: .github/workflows/purge-js-delivr-cache.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Curl"
+        uses: indiesdev/curl@v1.1
+        with:
+          url: https://purge.jsdelivr.net/
+          method: "POST"
+          accept: 200,201,204,202
+          headers: '{ "cache-control": "no-cache", "content-type": "application/json" }'
+          body: "{'path': ['/npm/@alma/widgets@3.x.x/dist/widgets.umd.js','/npm/@alma/widgets@3.x/dist/widgets.umd.js','/npm/@alma/widgets@3.x.x/dist/widgets.module.css','/npm/@alma/widgets@3.x/dist/widgets.module.css','/npm/@alma/widgets@3.x.x/dist/widgets.css','/npm/@alma/widgets@3.x/dist/widgets.css','/npm/@alma/widgets@3.x.x/dist/widgets.min.css','/npm/@alma/widgets@3.x/dist/widgets.min.css']}"
```
Comment (gautierlabarre) [inline .github/workflows/purge-js-delivr-cache.yml:16]: Do you think, there is a way to update the 3 with the current version ? I dunno how to do so easily

File: .github/workflows/purge-js-delivr-cache.yml (hunk 1 above)
Comment (FranceBe) [reply .github/workflows/purge-js-delivr-cache.yml]: Oops, totally

File: .github/workflows/purge-js-delivr-cache.yml (hunk 2 above)
Comment (FranceBe) [reply .github/workflows/purge-js-delivr-cache.yml:16]: I'm not sure how to get the current version from the CI, that can be an improvement for another PR, that is a a good idea

**Pull request merged**